### PR TITLE
Show people and vacancies in tree view

### LINKED ISSUE #484: Show people and vacancies in tree view
1) Add people view right below groups tree on a group page
2) Rename "Teams" tab to "Team"

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #484

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
Untitled 2

File: src/components/TeamPage.tsx [hunk 1]
```diff
@@ -36,6 +39,14 @@ export const TeamPage = ({ teamId }: TeamPageProps) => {
                         children.map((child) => <GroupTreeViewNode key={child.id} group={child} visible />),
                     )}
                 </TreeView>
+                <PreviewContent>
+                    {nullable(groupQuery.data, (g) => (
+                        <>
+                            <TeamPeople groupId={g.id} isEditable={true} />
```
Comment (asabotovich) [inline src/components/TeamPage.tsx:45]: Should it be always editable? 
For every user?

Comment (Katrin-kudryash) [reply src/components/TeamPage.tsx:45]: Oh, no

Comment (Katrin-kudryash) [reply src/components/TeamPage.tsx:45]: It`s hidden if user is`t an admin. Vadim added a component (```<Restricted visible={group.meta.isEditable}>``` ) that adjusts Visibility

#595

File: src/components/TeamPage.tsx [hunk 2]
```diff
@@ -36,6 +39,14 @@ export const TeamPage = ({ teamId }: TeamPageProps) => {
                         children.map((child) => <GroupTreeViewNode key={child.id} group={child} visible />),
                     )}
                 </TreeView>
+                <PreviewContent>
```
Comment (tsumo) [inline src/components/TeamPage.tsx:42]: You don't need PreviewContent here, it's not a preview.

Just add local styles if you need gaps between components

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/TeamPage.tsx [hunk 3]
```diff
@@ -17,6 +19,13 @@ interface TeamPageProps {
     teamId: string;
 }
 
+export const StyledWrappert = styled.div`
```
Comment (tsumo) [inline src/components/TeamPage.tsx:22]: `WrapperT`

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**